core/textile/client.go

```diff
@@ -18,19 +20,23 @@ import (
 	nc "github.com/textileio/go-threads/net/api/client"
 	bucketsClient "github.com/textileio/textile/api/buckets/client"
 	"github.com/textileio/textile/api/common"
+	uc "github.com/textileio/textile/api/users/client"
 	"github.com/textileio/textile/cmd"
 	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials"
 )
 
 type textileClient struct {
 	store            db.Store
 	threads          *threadsClient.Client
+	ht               *threadsClient.Client
 	bucketsClient    *bucketsClient.Client
 	isRunning        bool
 	Ready            chan bool
 	cfg              config.Config
 	isConnectedToHub bool
 	netc             *nc.Client
+	uc               UsersClient
```

Comment:
@jsonsivar 

Do you think it makes to make sense to make the names a bit more descriptive?

Reply:
I'm not too fussed about this but we can change it if we want. Thought `uc` made sense because it stands for user client and in the spirit of Go to keep things short :) (kinda like `netc`/`nc` right above it).